### PR TITLE
[Improvement] Preinst script for packages

### PR DESCRIPTION
**Pull request by maxmosk**
Partial solution #45 problem of preinst script

### CODE REVIEW

File: scripts/preinst.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if [[ $? == 0 ]]; then
+        editor=$item
+        break
+    fi
+done
+
+if [[ $editor == "NO EDITOR" ]]; then
+
+    echo "No supported DjVU viewer intalled"
+    exit 2
```
Comment (nerett) [inline scripts/preinst.sh]: Please check #41

File: scripts/install.sh [hunk 2]
```diff
@@ -5,65 +5,30 @@ home_dir="gnu-koryavov"
 scipts_dir="scripts"
```
Comment (nerett) [inline scripts/install.sh]: Please consider updating this file according to the upstream `install.sh`

File: scripts/preinst.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```
Comment (nerett) [inline scripts/preinst.sh]: Please consider updating this file according to the upstream `install.sh`

**Review by nerett — CHANGES_REQUESTED** [review]

File: scripts/install.sh (hunk 2 above)
Comment (maxmosk) [reply scripts/install.sh]: Ok, fixed

File: scripts/preinst.sh (hunk 3 above)
Comment (maxmosk) [reply scripts/preinst.sh]: If it was necessary to guarantee execution with bash, I fixed that. But if a different execution method was to be used there? Suppose python

File: scripts/preinst.sh (hunk 1 above)
Comment (maxmosk) [reply scripts/preinst.sh]: Ok, now it will only print message

File: configs/install.conf [hunk 4]
```diff
@@ -1,3 +1,7 @@
 editors=(okular atril evince xreader zathura)
```
Comment (nerett) [inline configs/install.conf]: `default_viewers` seems to be correct

File: scripts/preinst.sh (hunk 3 above)
Comment (nerett) [reply scripts/preinst.sh]: > updating this **file** ...

I meant `preinst.sh`

**Review by nerett — CHANGES_REQUESTED** [review]

File: configs/install.conf (hunk 4 above)
Comment (maxmosk) [reply configs/install.conf]: Agree, fixed

**Review by nerett — APPROVED** [review]

**Pull request merged**